FWCore/Catalog/interface/SiteLocalConfig.h

```diff
@@ -30,7 +57,9 @@ namespace edm {
     SiteLocalConfig() {}
     virtual ~SiteLocalConfig() {}
 
-    virtual std::vector<std::string> const& dataCatalogs(void) const = 0;
+    virtual std::vector<std::string> const& dataCatalogs() const = 0;
+    virtual std::vector<edm::CatalogAttributes> const& dataCatalogs_da() const = 0;
```

Comment:
I prefer to keep thing as is for now since if we will remove trivial data catalog in the future, there will be only one dataCatalogs(). The same comment is applied to ```_da``` postfix elsewhere. If we decide to still keep trivial data catalog code after turning on Rucio data catalog by default, the other way around should be used, I mean dataCatalogs() for Rucio and trivialDataCatalog() for Trivial. Please let me know if you think it is better to make the change right now, I mean dataCatalogs() for Rucio and trivialDataCatalogs() for Trivial.

Reply:
@makortel , how do you think about this?

Reply:
If the PR cleaning up the trivial catalog functionality and removing the `_da` suffix will be done quickly after this PR gets merged (even if we wouldn't merge it quickly), and will be backported to same set of earlier release cycles where this PR will be backported, I could accept the current naming. Otherwise I'd prefer to have a more self-documenting naming already here (`dataCatalogs()` and `trivialDataCatalogs()` would be fine).